src/patternfly/components/ContextSelector/examples/context-selector.css

```diff
@@ -5,3 +5,7 @@
 #ws-core-c-context-selector-with-footer {
   min-height: 380px;
 }
+
+#ws-core-c-context-selector-plain-with-text {
+  min-height: 380px;
```

Comment:
Since this is the same as the other menu heights, we can combine all of the selectors to one - 
```
#ws-core-c-context-selector-basic, #ws-core-c-context-selector-with-footer, #ws-core-c-context-selector-plain-with-text {
  min-height: 380px;
}
```